app/models/concerns/waste_exemptions_engine/can_copy_data_from_registration.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module WasteExemptionsEngine
+  module CanCopyDataFromRegistration
+    extend ActiveSupport::Concern
+
+    included do
+      after_initialize :copy_data_from_registration, if: :new_record?
+    end
+
+    def registration
+      @registration ||= Registration.find_by(reference: reference)
+    end
+
+    private
+
+    def copy_data_from_registration
+      attributes = registration.attributes.except("id",
```

Comment:
Does this mean we are copying the old reference to the RenewingRegistration? Is this likely to cause any issues when we complete it?

Reply:
No, I have removed the attribute from the one available for copying over :)

Reply:
Is used just for finding it back when we need it :)